examples/handlebars/components/notes/notes.spec.js

```diff
@@ -0,0 +1,23 @@
+const fractal = require('../../fractal.config.js');
+
+describe('tree', () => {
+    beforeEach(async () => {
+        await fractal.load();
+    });
+
+    it('loads notes from config', async () => {
```

Comment:
we should also look if a variant with notes that's defined in the config gets the proper notes.

Reply:
Done